README.md

```diff
@@ -56,9 +56,7 @@ _* Some How Can I code examples not included but topic introduced elsewhere._
 
 ## Reference Guides
 
-At this point in time, only `Python` bindings are available for `Polars`.
-
-> We are actively looking for `Node` experts to implement `JavaScript` bindings!
+At this point in time, only `Python` and `JavaScript` bindings are available for `Polars`.
```

Comment:
At this point in time, only `Python` and `NodeJS` bindings are available for `Polars`.

There are third party bindings btw, but idk if this is worth noting.